src/redux/basket-item-count/selectors.tsx

```diff
@@ -0,0 +1,2 @@
+export const selectBasketItemCount = (state) =>
```

Comment:
where do you use this? we decided to remove selectors at all

Reply:
removed